include/oneapi/dpl/experimental/kt/esimd_radix_sort_utils.h

```diff
@@ -578,33 +578,185 @@ template <typename T, int N, typename OpAlignedT = lsc_op_aligned_t<T>,
           int NElts = lsc_op_block_size<T, N, OpAlignedT>(),
           ::std::enable_if_t<NElts != lsc_op_block_size_rounding<NElts>(), int> = 0>
 void
-BlockStore(uint32_t slm_offset, __dpl_esimd_ns::simd<T, N> data)
+BlockStoreSlm(uint32_t slm_offset, __dpl_esimd_ns::simd<T, N> data)
 {
     constexpr int BLOCK_SIZE_ROUNDED = lsc_op_block_size_rounding<NElts>();
 
     constexpr int BLOCK_SIZE = lsc_op_block_size<OpAlignedT, BLOCK_SIZE_ROUNDED, T>();
-    BlockStore<T, BLOCK_SIZE>(slm_offset, data.template select<BLOCK_SIZE, 1>(0));
-    BlockStore<T, N - BLOCK_SIZE>(slm_offset + BLOCK_SIZE * sizeof(T),
-                                  data.template select<N - BLOCK_SIZE, 1>(BLOCK_SIZE));
+    BlockStoreSlm<T, BLOCK_SIZE>(slm_offset, data.template select<BLOCK_SIZE, 1>(0));
+    BlockStoreSlm<T, N - BLOCK_SIZE>(slm_offset + BLOCK_SIZE * sizeof(T),
+                                     data.template select<N - BLOCK_SIZE, 1>(BLOCK_SIZE));
 }
 
-template <typename T, int N, __dpl_esimd_ens::cache_hint H1 = __dpl_esimd_ens::cache_hint::none,
-          __dpl_esimd_ens::cache_hint H3 = __dpl_esimd_ens::cache_hint::none>
-inline std::enable_if_t<(N * sizeof(T) <= 256), void>
-BlockStore(T* dst, __dpl_esimd_ns::simd<T, N> data)
+template <typename T, int NElts, ::std::enable_if_t<sizeof(T) == sizeof(::std::uint8_t), int> = 0>
+constexpr int
+lsc_block_store_size_rounding()
```

Comment:
You can rename it to e.g. `lsc_op_block_size_rounding`, because these limitations are the same for both store and load operations.

Reply:
The old functions can then be renamed to `lsc_slm_op_block_size` and `lsc_slm_op_block_size_rounding`

Reply:
The existing `BlockLoad` can be renamed to `BlockLoadSLM`

Reply:
Done: renamed to ``BlockLoadSlm```